AEA-1385 fipa end2end test for python and go (#2464)
fetchai/agents-aea: libs/go/aea_end2end/README.md

# End2end example Go <> Python

## Run example

> ensure all dependencies are installed: python, aea, golang

Reviewer: ```suggestion
Ensure all dependencies are installed: python, aea and golang.
```

> run to launch buyer agent

Reviewer: ```suggestion
To launch the buyer agent run:
```

`./run_buyer.sh`

> run in another terminal to launch seller agent 

Reviewer: ```suggestion
In another terminal, to launch the seller agent run:
```

`./run_seller.sh`

> after a while both agents get connected to p2p fetchai network,
> perform fipa message exchange and show `FIPA INTERACTION COMPLETE` in output logs
> 
> terminate every agent with `ctrl+c`

Reviewer: ```suggestion
After a while both agents get connected to the ACN, perform a `fetchai/fipa` message exchange and show `FIPA INTERACTION COMPLETE` in output logs.

Terminate every agent with `ctrl+c`.
```

## Generate protocol

> 

Reviewer: ```suggestion

To generate a protocol, use the following approach:

```

``` bash
aea create temp_agent
cd temp_agent
aea generate protocol -l PATH_TO_SPEC

> ```

Reviewer: ```suggestion
```

Provide additionally `--l go` to generate the go files.
```
